docs/latex.rst

```diff
@@ -256,3 +256,37 @@ A macro that resolves to the current repository URL
 
 A macro that resolves to the current commit SHA
 (i.e., ``31860f2f558b05d8c941d8f73c64f5dbf5ee79db``).
+
+
+Reproducibility paragraph
+-------------------------
+
+Many scientific journals (e.g., those of AAS) will require the authors
```

Comment:
I'd say that it's less about the journals "requiring" it (although that may be a factor) and more about the fact that we (as the authors) want to share info about the mode we're working in. So I might re-phrase this intro...